### ISSUE TITLE
Unable to run ansible-inventory-grapher --version after install

### ISSUE THREAD

**Issue by tchia04** [willthames/ansible-inventory-grapher#24]
After installing using sudo pip install ansible-inventory-grapher on a Mac, I could not even run ansible-inventory-grapher --version or --help

I am using ansible version 1.9.4

I got the following error message. Is there some dependencies missing? 
<pre>
ansible-inventory-grapher --help
Traceback (most recent call last):
  File "/usr/local/bin/ansible-inventory-grapher", line 23, in <module>
    from ansible.cli import CLI
ImportError: No module named cli
</pre>

**Comment by willthames** [comment]
Sorry about that - ansible-inventory-grapher should work with Ansible 1.9 still. I've fixed the recent vault improvements so that they cope with Ansible 1.9 too. 

I've also ensured the very minimal tests still work, but we don't have coverage of the CLI usage at all. 

```
ansible-inventory-grapher -i test/vault/inventory web-01
ansible-inventory-grapher --vault-password-file=test/vault/vaultpass -i test/vault/inventory web-01
```

work under both Ansible 1.9 and Ansible 2.0+ now.

I've pushed 2.3.1 to address this.
